applications/plugins/SofaMiscCollision/SofaMiscCollisionConfig.cmake.in

```diff
@@ -3,6 +3,15 @@
 @PACKAGE_GUARD@
 @PACKAGE_INIT@
 
+find_package(SofaBaseCollision QUIET REQUIRED)
```

Comment:
what does it means again ```QUIET```  and ```REQUIRED``` ?

Reply:
It tells to CMake "I need this package so you must fail if you can't find it (REQUIRED), but I don't want to see that you found it in the logs (QUIET)".